src/sentry/integrations/github/search.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+from sentry.api.bases.organization import OrganizationEndpoint, OrganizationPermission
+from sentry.models import Integration
+
+
+class GitHubSearchEndpoint(OrganizationEndpoint):
+    permission_classes = (OrganizationPermission, )
+
+    def get(self, request, organization, integration_id):
+        try:
+            integration = Integration.objects.get(
+                organizations=organization,
+                id=integration_id,
+                provider='github',
+            )
+        except Integration.DoesNotExist:
+            return Response(status=404)
+
+        field = request.GET.get('field')
+        query = request.GET.get('query')
+        if field is None:
+            return Response({'detail': 'field is a required parameter'}, status=400)
+        if not query:
+            return Response({'detail': 'query is a required parameter'}, status=400)
+
+        installation = integration.get_installation(organization.id)
+        if field == 'externalIssue':
+            repo = request.GET.get('repo')
+            if repo is None:
+                return Response({'detail': 'repo is a required parameter'}, status=400)
+
+            try:
+                response = installation.search_issues(
```

Comment:
Should this be using `search_repositories`?

Reply:
this should just be searching issues within a certain repo

Reply:
Ah I see now. This searches multiple things